_posts/2018-09-13-is-cockpit-secure.md

```diff
@@ -0,0 +1,255 @@
+---
+title: Is Cockpit Secure?
+date: 2018-09-13 10:00
+category: blog
+tags: cockpit
+slug: is-cockpit-secure
+---
+
+Security is of your Linux systems vital. It's not a binary thing though.
```

Comment:
The wording of the first sentence seems strange.  Perhaps:

  "Security is vital to your Linux systems."

instead?

Reply:
The Yoda talk fixed I have. Thanks for spotting!